### ISSUE TITLE
Requesting location permissions

### ISSUE THREAD

**Issue by ahaverty** [SchibstedSpain/Leku#81]
v3.1.0

I'm using Leku as a fallback to allow users to select their location if it wasn't found or if they deny location permissions in a previous activity.

**Right now my flow is:**
MyActivityNeedingLocationExample.java
```
if permissions granted: use location/fused and continue without Leku
if permissions denied: start Leku
```

It seems that Leku always ask for permissions on start. I couldn't see any documentation on preventing this. It would be great if that was possible, or if anyone knows whether a PR to fix would need a lot of changes? Thanks

**Comment by ferranpons** [comment]
@ahaverty It will be solved in the next release. Thanks for contributing!

**Comment by ferranpons** [comment]
Solved with version 3.2.0 of the library.
